examples/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "logging": {
+    "logs": {
+      "default": {
+        "exclude": [
+          "http.log.access.log0"
+        ]
+      },
+      "log0": {
+        "writer": {
+          "output": "stdout"
+        },
+        "include": [
+          "http.log.access.log0"
+        ]
+      }
+    }
+  },
+  "apps": {
+    "supervisor": {
+      "supervise": [
+        {
+          "command": ["php-fpm"],
+          "dir":  ".",
+          "env": ["ADDITIONAL_ENV=1"],
```

Comment:
I wonder if this would be more intuitive as a JSON object (map[string]string)
```
"env": {
  "ADDITIONAL_ENV": "1"
}
```
What do you think? Would it bring some complications to parse, or to show in caddy documentation?

Reply:
👍 I just copied what v1 options was expecting, but it would be much better like this :)